governance/roles.md

```diff
@@ -96,9 +96,42 @@ TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/maste
 
 ## Facilitation roles
 
-Facilitation roles are identified in [github settings](/.github/settings.yaml)
-and include additional permissions in the repo which are governed by policies
-described below.
+Members often contribute by working in small groups that research and discuss
+options and then share their findings with the rest of the group in a
+presentation and typically contribute to the body of work in this
+repository (via Pull Request). Some tasks can be effectively executed
+by simply chiming in on a github issue and independently contributing a PR
+without any particular role or authorization.
+
+This section describes roles where more coordination is helpful and the
+work benefits from identifying one or more Members who will take
```

Comment:
Notice the capitalization of "members" on line 107 and 108, is this intentional?  if so we need to ensure we are consistent in our other docs too.

Reply:
made it consistent with rest of doc that "member" is lower case.  (Note that "role" capitalization is inconsistent, but don't want to expand the scope of this PR)

Reply:
wrote up separate issue for capitalization consistency of Roles and Chair https://github.com/cncf/sig-security/issues/231